src/Home/components/LecueBookList/LecueBookList.style.ts

```diff
@@ -0,0 +1,61 @@
+import styled from '@emotion/styled';
+
+export const LecueBookListWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+
+  width: 100%;
+`;
+
+export const Title = styled.header`
+  width: 100%;
+  padding: 1.5rem 0;
+
+  border-color: ${({ theme }) => theme.colors.BG};
+  border-width: 0.1rem 0;
+  border-style: solid;
```

Comment:
p2: 요 부분을

```suggestion
  border: 0.1rem solid ${({ theme }) => theme.colors.BG};
```
로 간단하게 줄여볼 수 있을 것 같아!